syntax = "proto3";

package helloworld;

option go_package = "idl/grpc/helloworld";

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
  string name = 1;
}

message HelloResponse {
  string message = 1;
}
